microsoft-edge/webview2/concepts/webview-features-flags.md

```diff
@@ -114,6 +114,7 @@ The following are some of the flags we've seen used.
 | `msEdgeDesignerDriverFix ` |  Use this flag to disable getting content and automatically showing the Designer feature. This feature is not supported in WebView2. |
 | `msEdgeFluentOverlayScrollbar` | Force-enables Fluent Overlay scrollbars, overriding whatever value `kFluentScrollbar` may hold at the moment. |
 | `msEdgeDevToolsWdpRemoteDebugging` | Enables remote debugging.  See [Remote debugging WebView2 WinUI 2 (UWP) apps](../how-to/remote-debugging.md), [Remotely debug Windows devices](../../devtools-guide-chromium/remote-debugging/windows.md). |
+| `msEdgeWebViewApplyWebResourceRequestedFilterForOOPIFs` | If this flag is enabled, `AddWebResourceRequestedFilter` will also apply to out-of-process iframes. <br/>.NET: [AddWebResourceRequestedFilter](/dotnet/api/microsoft.web.webview2.core.corewebview2.addwebresourcerequestedfilter) <br/>WinRT: [AddWebResourceRequestedFilter](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2#addwebresourcerequestedfilter) <br/>Win32: [AddWebResourceRequestedFilterWithRequestSourceKinds](/microsoft-edge/webview2/reference/win32/icorewebview2_22#addwebresourcerequestedfilterwithrequestsourcekinds) |
```

Comment:
does Win32 have a `AddWebResourceRequestedFilterWithRequestSourceKinds` method?  I only find a `AddWebResourceRequestedFilterWithRequestSourceKinds` method, by Searching Ref docs and by Find in "Overview of APIs".

present article section rendered:
https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/webview-features-flags?branch=pr-en-us-3275#available-webview2-browser-flags - Find msEdgeWebViewApplyWebResourceRequestedFilterForOOPIFs

Overview of APIs, rendered:
https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-3275&tabs=win32cpp#manage-network-requests-in-webview2 - find AddWebResourceRequestedFilter

@monica-ch @nishitha-burman

Reply:
Win32 has only AddWebResourceRequestedFilterWithRequestSourceKinds